.vscode/settings.json

```diff
@@ -9,14 +9,6 @@
   "yaml.schemaStore.enable": true,
   "yaml.validate": true,
   "yaml.schemas": {
-    "http://json-schema.org/draft-07/schema#": "*.schema.yaml",
```

Comment:
this is the 2nd time you're deleting these other schemas. Undo the deletion of these other schemas.

Reply:
These are settings for all people who clone repo. I would like for the default settings to point to something that is working right now. We can switch to different one when other changes are finalized.

Reply:
exactly my point here. There are two schemas in this repo, this settings.json file was configured to support both of them, along with the yaml schema files.
Bring them back so you do not break other users of this repo.

Reply:
Please see my other comment which suggests a different config which should support what you are needing along with keeping the existing stuff working too.